### ISSUE TITLE
Meta search should be disabled on all content results

### ISSUE THREAD

**Issue by panchovm** [WordPress/openverse-frontend#1585]
## Description
In all content results, the meta search section is visible at the end of the grid. This should be disabled as the section was designed to list sources related to a single content type and not multiple, as in this layout case. This logic applies to the outdated and the new design.

## Reproduction
1. Search for something.
2. Go or switch to All content results.
3. See the final section, after the results grid.
4. See the meta search section.

## Screenshots
<img width="1000" alt="CleanShot 2022-07-26 at 09 43 45@2x" src="https://user-images.githubusercontent.com/895819/181020944-6ff8ef7d-e362-4400-9b28-0dd01dcb5bdd.png">

## Resolution
- [ ] 🙋 I would be interested in resolving this bug.

**Comment by zackkrida** [comment]
@panchovm I'm concerned about hiding this feature form our default search view. 

What do you think if we showed multiple dropdowns, one for each content type, or a single dropdown grouped by "image", "audio" and so on?

**Comment by panchovm** [comment]
Why are you concerned? I think that showing the metasearch forces this layout unnecessarily. The external sources list is for visitors not finding what they want, and that makes more sense doing it on the content type result after loading several results.

I am curious to hear other thoughts.

**Comment by ramadanomar** [comment]
![nometasearch](https://user-images.githubusercontent.com/55840806/185667955-3cd3e2b9-1442-4dd0-9bbe-c82389e8053a.png)
For further context, this is how it would look if we removed the metasearch on <all> searches. I don't think it disturbs anyone, especially because it is located after the results.

**Comment by ramadanomar** [comment]
I agree with @zackkrida that we should add a more generic search text.
Currently we have it hard coded to display images for all content. 
```js
    const metaSearchFormType = computed(() => {
      if (mediaTypes.includes(props.searchType)) {
        return props.searchType
      }
      return IMAGE
    })
```

**Comment by panchovm** [comment]
Not sure if I understood your point @ramadanomar correctly. Are you agree showing or hiding the metasearch section?

**Comment by ramadanomar** [comment]
Any update on this issue?

**Comment by zackkrida** [comment]
My thinking here, @panchovm was that I don't like conditionally showing/hiding pieces of the UI without the user knowing about it. 

A similar concern I have is in the filter sidebar. That might be a good example. I plan to make an issue for this today, but I do not like how the filter sidebar has fewer filters on the 'all results' view than the single media type views. Of course, this is necessary, but I think it would be great to have a message at the bottom of the filterbar that says something like 'Choose a single media type to view detailed source and media-specific filters', something like that. 

My logic here is the same. I don't want 'all results' to be an inadequate experience if it doesn't have to be.

**Comment by panchovm** [comment]
> I don't like conditionally showing/hiding pieces of the UI without the user knowing about it.

I didn't understand this point, but I understand the core of your concern. If we think is useful, then let's tag it with `needs design` to design a solution.

> I do not like how the filter sidebar has fewer filters on the 'all results' view than the single media type views.

Showing more or fewer filters should not be a concern as far as they relate to the result on the left. I have discussed this several times with other designers when designing the header interaction and tabs for content type/filters. Showing filters that impact just a portion of the results, like `aspect ratio` for images or `duration` for audio, is risky and could be very confusing.

> My logic here is the same. I don't want 'all results' to be an inadequate experience if it doesn't have to be.

Fully agree with you. And because of that, I need to insist on an argument mentioned in previous discussions: if we do not know how visitors value and interact with this page, then we should not add more features.

**Comment by zackkrida** [comment]
> Showing filters that impact just a portion of the results, like aspect ratio for images or duration for audio, is risky and could be very confusing.

I agree with this and definitely do not want to do it. I just want users to know that more filters are available. That's my core concern: if we have features that are only available in certain views, that aren't the default view, how can we make sure users know about them?

> if we do not know how visitors value and interact with this page, then we should not add more features.

I'm not sure I agree with this. I definitely want user data to inform our decisions as much as possible, but there occasions where we have to use our expertise to make a best-guess. For example, we have _no way of knowing_ how many Openverse users use screenreaders. Does that mean we shouldn't make things accessible? I don't think so, because accessibility is something we value and _think is_ valuable to users, even if its a small subset of users.

I think we can work towards getting more data _and_ make best-guess desicions using our judgement at the same time.

**Comment by marko-srb** [comment]
Hi there. My intuitive thoughts are: 

Having 'Metasearch' is a good thing.

After researching at Openverse, I'd need additional resources to find the appropriate content. Way to have it, while not overwhelming the user, in my opinion, is:
01. Have a dropdown or hover list of external resources, below the grid.
02. Have that dropdown or hover list only on specific (one type) content results and not in general search where all content results appear.

To expand. 
01. Dropdown or hover list will condense the information and not confuse the user. Although an additional step, the user will happily hover or click when they need the content. 
02. Why shouldn't 'All content results' include metasearch?
We can confidently assume that content research goes like any other research. First, you research your topic very broadly, then narrow it down and focus on one thing at a time. The same shall apply to 'Openverse' or any other search engine. Once I find what I am seeking about my topic, I'll focus only on image results to find the image I need to use. Or audio. Or other. And once I am in the image results only, I may be thankful to learn about different resources ('Metasearch') if the results I see in Openverse are not what I need. 

Additionally.
*And what about it being at the end of the page? Well, we don't need to suggest to the user that we have a helpful 'Metasearch' with external resources, we want them to use Openverse. Only if they don't find what they need, we want to be valuable enough and leave metasearch afterward. So, underneath the grid is a good place for me.

Hope this helps.

**Comment by jasmussen** [comment]
Great thoughts all around! Regardless of when/where/how the meta-search tool shows up, I would echo the opportunity to minimize its design to have less prominence, which could in turn make feature more glancable/actionable.

Whether it shows up only on individual filters, or even on the "all content search" I have a less strong opinion about. But it's worth noting that even if it does show up everywhere, the content of the dropdown will still change, as it doesn't make sense to suggest "soundcloud" on the Image filter, for example.

**Comment by panchovm** [comment]
The Metasearch is definitely a great feature. It allows visitors to find relevant content when they do not find something in Openverse.

I agree with you @marko-srb that it makes more sense to show the external sites in the single-content results as it is consistent with the content seeking. The dropdown design idea was already designed, but its implementation in #734 is blocked by a shift in the term used for the sources in #1197.

Your point @jasmussen aligns with mine about adding a browsing complexity to the meaning of this page. If we go with keeping the feature, it would be necessary to split the dropdown and create a new variant of the menu.

Thanks for jumping in and sharing your thoughts.

**Comment by panchovm** [comment]
> Hiding it on the 'all results' view and unblocking this

I like this idea. I am putting down the questions we want to respond to with user data. To continue, I will remove the `needs design` label and @ramadanomar can continue as he [was interested](https://github.com/WordPress/openverse-frontend/issues/1585#issuecomment-1216804697) in taking this issue.
